src/arithmetic/arithmetic_expression.h

```diff
@@ -139,6 +146,8 @@ void AR_EXP_ToString(const AR_ExpNode *root, char **str);
  * func - function name to lookup. */
 bool AR_EXP_ContainsFunc(const AR_ExpNode *root, const char *func);
 
+AR_ExpNode *AR_EXP_SeekFunc(AR_ExpNode *root, const char *func);
```

Comment:
Please document this new function, see if we can't what we already have instead of adding new functions to the current `AR_EXP` API